packages/xo-web/src/common/notification.js

```diff
@@ -32,39 +32,37 @@ export class Notification extends Component {
     return false
   }
 
-  render() {
-    return (
-      <ReactNotify
-        ref={notification => {
-          if (!notification) {
-            return
-          }
+  _addNotifications = notification => {
+    if (!notification) {
```

Comment:
explicit your test, maybe `notification == null` !

Reply:
I avoid the callback with 'null' so I don't need test.

Reply:
I understand now :) I approve these changes